### DISCUSSION TITLE
How to separate policies when one server and multiple clients in strongwan

### DISCUSSION THREAD

**Discussion by kodumulo** [strongswan/strongswan#1767] [Q&A]
Hello?

We have one policy on one server, and several clients are stuck together and in use (ubuntu

In this situation, I would like to separate the policy on the server by client. We are using the authentication method based on the certificate, and it is difficult to issue individually.

Therefore (remote.id , loca.id not available)

The client is also receiving eap-radius authentication from the server.

At this time, the eapid is different for this time,
It failed when I tried to separate it with this.

Based on what can we separate policies in this situation?

* Why separate policies
Previously, it was controlled through iptabels with vip band
Because now I want to control it through the children firewall included inside the swan.

Each client has the same server that you want to communicate with
Because the ports and services allowed are different for each client
I want to separate it.

**Comment by kodumulo** [comment]
@tobiasbrunner

**Reply by tobiasbrunner** [reply]
Sorry, I don't really understand what you are asking. Since you are mentioning RADIUS you may want to look into [groups/class attributes](https://docs.strongswan.org/docs/5.9/plugins/eap-radius.html#_group_selection).
